src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx

```diff
@@ -183,30 +184,34 @@ export class ConnectedFilterControl extends React.Component<
     }
     return actionsToReturn;
   }
-}
 
-function popoverFrom(actions: PopoverableAction[]): React.ReactElement<any>[] {
-  return actions.map((action) => {
+  private activatorButtonFrom(action: BaseAction): React.ReactElement<any> {
     return (
-      <Item key={action.key}>
-        <Popover
-          active={action.popoverOpen}
-          activator={activatorButtonFrom(action)}
-          onClose={action.onAction}
-          preferredAlignment="left"
-          sectioned
-        >
-          {action.popoverContent}
-        </Popover>
-      </Item>
+      <Button
+        onClick={action.onAction}
+        disclosure
+        disabled={this.props.disabled}
+      >
+        {action.content}
+      </Button>
     );
-  });
-}
+  }
```

Comment:
Is this to stop them from clicking on popover elements if the filter gets disabled while the popover is already open? If so could we just close the popover if the filter is disabled instead?